bin/git-coauthor

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+coauthor=$1
+email=$2
+
+test -z $coauthor && echo "co-author required." 1>&2 && exit 1
+test -z $email && echo "co-author email required." 1>&2 && exit 1
+
+old_message="$(git log --format=%B -n1)"
```

Comment:
After running this command twice on the same commit, I found there is a newline between two `Co-authored-by`.
```
commit 698b33b589fdd3d05dc5f0d9912228a78d9a9537 (HEAD -> master)
Author: spacewander <spacewanderlzx@gmail.com>
Date:   Sat Aug 24 11:03:18 2019 +0800

    init
    
    Co-authored-by: John Doe <john@doe>
    
    Co-authored-by: John <john@doe>
```

It's caused by a trailing newline in the output of `git log --format=%B -n1`.
Better to find a way to remove the annoying newline.

Reply:
Maybe we can remove the trailing newline when the original commit message already contains `Co-auhored-by`.